build-routers.sh

```diff
@@ -78,6 +79,23 @@ function retrieveHsl() {
   cp hsl.zip $ROUTER_FINLAND
 }
 
+function retrieveJoensuu() {
+  echo "Retrieving Joensuu data..."
+  cd $ROUTER_JOENSUU
+  curl -sS "http://dev.hsl.fi/gtfs.waltti/joensuu.zip" -o joensuu.zip
+  curl -sS "http://dev.hsl.fi/gtfs.waltti/posjoe.zip" -o posjoe.zip
```

Comment:
Did you mean that  GTFS zip retrieval for all sources should be moved to Dockerfile?